app/views/contexts/summary/group.py

```diff
@@ -1,15 +1,19 @@
-from typing import Iterable, Mapping, MutableMapping
+from typing import Iterable, Mapping, MutableMapping, Type
 
 from werkzeug.datastructures import ImmutableDict
 
 from app.data_models import AnswerStore, ListStore, ProgressStore
 from app.data_models.metadata_proxy import MetadataProxy
 from app.questionnaire import Location, QuestionnaireSchema
 from app.questionnaire.placeholder_renderer import PlaceholderRenderer
+from app.questionnaire.questionnaire_schema import is_list_collector_block_editable
 from app.survey_config.link import Link
 from app.views.contexts.summary.block import Block
 from app.views.contexts.summary.calculated_summary_block import CalculatedSummaryBlock
 from app.views.contexts.summary.list_collector_block import ListCollectorBlock
+from app.views.contexts.summary.list_collector_content_block import (
+    ListCollectorContentBlock,
+)
 
 
 class Group:
```

Comment:
As Mebin commented on the looping pr, we've now got a gap where a CalculatedSummary of repeating questions from a ListCollectorContent block won't work, I think it just needs a check similar to below for which class to initialise the `list_collector_block` with on line 103, and the `get_repeating_block_related_answer_blocks` to be on the base block class

However as this needs testing and probably another schema, maybe it should be separated out onto another card?

Reply:
Its on `ListCollectorBlock` but I think it needs to be on `ListCollectorBaseBlock` so that `ListCollectorContent` can also make use of it

Reply:
Added now so it's also `ListCollectorBlock` ones, thanks. `get_repeating_block_related_answer_blocks` moved to base block now.